modules/dateTime.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] `required`
- [ ] amazing job so far 👏 
- [ ] Please enable the disabled linters cause it defeats the purpose of installing linter 🥂